ApiFirst/ApiFirst/RepositoryList.cs

```diff
@@ -0,0 +1,180 @@
+﻿using DomainModels.DataBase;
+using DomainModels.Models;
+using DomainServices.Interfaces;
+
+namespace DomainServices.Repositories
+{
+    public class RepositoryList : ICustomerRepository
+    {
+        public int Create(Customer model)
+        {
+            model.Cpf = model.Cpf.Trim().Replace(".", "").Replace("-", "");
+            int response = 409;
+
+            ListCustomer list = ListCustomer.GetInstance();
+
+            model.Id = list.GetCustomerList().Count;
+
+
+            if (!list.GetCustomerList().Any())
+            {
+                model.Id = 0;
+                GetAll().Add(model);
+                return 201;
+            }
+
+            foreach (Customer item in list.GetCustomerList())
+            {
+                if (model.Cpf != item.Cpf && model.Email != item.Email)
+                {
+                    GetAll().Add(model);
+                    response = 201;
+                }
+                return response;
+            }
```

Comment:
Sua lógica de auto incrementar IDs está meio quebrada

Exemplo:
Lista com os IDs: 1, 2, 3, 4; count = 4,
Adicionar novo customer: next id = count + 1 = 5
Resultado = 1, 2, 3, 4, 5
Até aí tudo bem, irá adicionar um novo ID sem problemas

Se eu deletar um customer e depois adicionar um novo, ficará assim: (Customer deletado: id = 1)
Lista com os IDs: 2, 3, 4, 5; count = 4;
Adicionar novo customer: next id = count + 1 = 5
Resultado = 2, 3, 4, 5, 5
O ID = 5 se repete

Se eu deletar novamente um customer e depois adicionar um novo... (Customer deletado: id = 2)
Lista com os IDs: 3, 4, 5, 5; count = 4;
Adicionar novo customer: next id = count + 1 = 5
Resultado = 3, 4, 5, 5, 5
O ID = 5 se repete

Resultado Final => IDs: 3, 4, 5, 5, 5; count = 5
Não podemos repetir ID, pois é um identificador individual.
```suggestion
  model.Id = list.LastOrDefault()?.Id + 1 ?? 1;
```